Feat/27/fix dtos

stksl opened the pull request.

stahkravtsov committed:
fixed bug with user roles

stahkravtsov committed:
Merge branch 'dev' into feat/25/jwt-token

stahkravtsov committed:
added user secret

WesTer006 committed:
fixed session and movie deletion

stahkravtsov committed:
Updated Db

stahkravtsov committed:
remove unused usings

WesTer006 committed:
add hall-controller

WesTer006 committed:
commit before merging

stahkravtsov committed:
unified using of dto and models

stahkravtsov committed:
refactoring

WesTer006 committed:
added SeatService but need changes

WesTer006 committed:
resolved conflicts before merging

WesTer006 committed:
bug-fix after merging

stahkravtsov committed:
Added actors controller, movie controller without external id exist check

WesTer006 committed:
fix hall and seat controller need some changes in working with ID

stahkravtsov committed:
Complate content conrollers

WesTer006 committed:
commit before merging with branch feat/25/auth-access

WesTer006 committed:
added ticket controller, need to add mapping CreateTicketDTO->TicketDTO

stksl committed:
business logic and naming fixes

stksl committed:
fixes v2

stksl committed:
added validations

stksl committed:
resolved conflicts

stksl committed:
exception handling and ticket entity

added exception handling and removed IsBooked from Ticket entity

stksl committed:
exception handler with ticket migration + fixes

stksl committed:
fixes

stksl committed:
final fixes

stahkravtsov reviewed (changes requested):
Please, change naming for private methods to Pascal case (private void CheckHall(...) {} for example), and add validators for all DTO we can receive from user

stksl committed:
minor fixes

added login validator, private methods to PascalCase and unified badrequest payload

stksl committed:
http url validation

stksl committed:
ticket fix
